src/ate/test_programs/cp.cc

```diff
@@ -125,13 +124,31 @@ int main(int argc, char **argv) {
     LOG(ERROR) << ate_opts_result.status().message() << std::endl;
     return -1;
   }
+  // Validate FPGA bitstream path.
   AteClient::Options ate_options = ate_opts_result.value();
-  auto dut_opts_result = ValidateDutOptions();
-  if (!dut_opts_result.ok()) {
-    LOG(ERROR) << dut_opts_result.status().message() << std::endl;
+  auto bistream_result = ValidateFilePathInput(
+      absl::StrCat("third_party/lowrisc/ot_bitstreams/cp_",
+                   absl::GetFlag(FLAGS_fpga), ".bit"));
```

Comment:
In general, I don't like hardcoded paths like this in programs, but I see the convenience factor you're going for.

I don't want to over-engineer this, but if you'd like to get rid of the hardcoded path and still have convenience, I'd try something like this:

```
ABSL_FLAG(std::string, bitstream, "third_party/lowrisc/ot_bitstreams/cp_$fpga.bit", "Bitstream to load");

...

auto bitstream_result = ValidateFilePathInput(
    absl::StrReplaceAll(absl::GetFlag(FLAG_bitstream), {
        {"$fpga", absl::GetFlag(FLAGS_fpga)}
    }));
```

You can really use any pattern for the substitutions (e.g. `{fpga}` instead of `$fpga`).